crates/cmd/src/main.rs

```diff
@@ -166,68 +143,80 @@ enum Subcommands {
     #[arg(short, long, value_enum)]
     guide: Guide,
     /// The certificate as a DER or PEM encoded file.
-    #[arg(short, long)]
-    path: PathBuf,
+    paths: Vec<PathBuf>,
```

Comment:
Override the argument to convey that it is a variadic parameter. That way it doesn't read `paths ...` but instead `file ...` to show that it takes one or more files/paths (see [`rm(1)`](https://linux.die.net/man/1/rm)).

```suggestion
    #[clap(value_name = "FILE")]
    paths: Vec<PathBuf>,
```

Reply:
Done.